Migrating-AU-to-Chocolatey-AU.md

```diff
@@ -0,0 +1,27 @@
+# Migrating From AU To Chocolatey AU
+
+With the [`1.0.0` release] of Chocolatey AU, there is a need to migrate to the new package name.

```

Comment:
If we will be going forward with reference linking (based on Paul's comment, we won't though), let's use it according to specs.
The specs define them as needing to have two square brackets, so in this case it would be

``[`1.0.0` release][]`` (the first one is the text being displayed, and the second being the name of the link which will use the text if it is empty).